### ISSUE TITLE
Running just "mite" results in an error

### ISSUE THREAD

**Issue by fstoerkle** [Overbryd/mite.cmd#17]
If i run just $> mite
than the following error occurs.
The other functions of mite work properly.

```
/opt/local/lib/ruby/gems/1.8/gems/Overbryd-mite.cmd-0.1.9/bin/../lib/../vendor/yolk-mite-rb-0.0.3/lib/mite/tracker.rb:6:in `current': undefined method `[]' for nil:NilClass (NoMethodError)
from /opt/local/lib/ruby/gems/1.8/gems/Overbryd-mite.cmd-0.1.9/bin/../lib/mite_cmd/application.rb:83:in `run'
from /opt/local/lib/ruby/gems/1.8/gems/Overbryd-mite.cmd-0.1.9/bin/../lib/mite_cmd.rb:30:in `run'
from /opt/local/lib/ruby/gems/1.8/gems/Overbryd-mite.cmd-0.1.9/bin/mite:9
from /opt/local/bin/mite:19:in `load'
from /opt/local/bin/mite:19
```

I use Ruby 1.8.7 (2009-06-08 patchlevel 173) [i686-darwin9] from MacPorts on OS X 10.5.7.

**Comment by Overbryd** [comment]
Thanks for submitting this issue.

I've tried but I couldn't rebuild this error. I'll try some more things but at the moment I have no clue why there is beeing returned nil.

**Comment by groenewege** [comment]
I encounter the same error.
maybe this information is relevant : 
http://stackoverflow.com/questions/5176782/uninitialized-constant-activesupportdependenciesmutex-nameerror

**Comment by Overbryd** [comment]
Hey everybody,

I'm really sorry not having looked into the issue for _quite_ a while now.
It seems the current mite.cmd is far out of date. That is why I've decided to do a rewrite, plus improving stability. Back then, this was my first command line tool I wrote. Now, I learned a lot more and when looking at the code of mite.cmd I get the yikes...

So feel free to open some `feature` issues if you'd like to see a particular feature in the rewrite.
